ITKIOScanco_UnitConversion.ipynb

```diff
@@ -0,0 +1,247 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.    HU_values = [HU_pixel_low, HU_pixel_high]
</pre><p>Please print out all the relevant scalar values here at the end of the cell and save the output in the cell so people browsing the notebook online can see examples of typical values.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/ITKIOScanco/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>